README.md

```diff
@@ -107,4 +107,15 @@ tests/test_client.py .                                                    [ 25%]
 tests/test_viewmodel.py ...                                               [100%]
 
 ============================== 4 passed in 0.33s ===============================
-```
\ No newline at end of file
+```
+
+## Deployment
+
+The app makes used of GitHub Actions to:
+- On push, build and run the 'test' image
+- On push to master, build the production image, push to docker repo, deploy to Azure Container
+
+Azure Container Instance URL: http://pdc-todo-app.uksouth.azurecontainer.io:8000/
```

Comment:
You will need to add a link to the Docker Hub image you created here.
As per the submission checklist **"Please include a link in your README"**

Reply:
Additionally you will need to add the Docker image deployment process.
As per the submission checklist **"Your Readme documents your manual deployment process, including the steps to release the Docker image, but doesn’t expose the webhook itself!"**

Reply:
Link to Docker Hub repo now in readme.

The ## Deployment section of the readme describes the GitHub actions for build, test and deploy.

There is no webhook involved.